kv/task.go

```diff
@@ -863,12 +815,6 @@ func (s *Service) deleteTask(ctx context.Context, tx Tx, id influxdb.ID) error {
 		return influxdb.ErrUnexpectedTaskBucketErr(err)
 	}
 
-	if err := s.deleteUserResourceMapping(ctx, tx, influxdb.UserResourceMappingFilter{
```

Comment:
We don't create URMs anymore for tasks (like this anyway). So we don't need this.
This _may_ leave some dangling URMs, but they are harmless cruft.